hw/ip/otp_ctrl/doc/_index.md

```diff
@@ -390,10 +390,15 @@ A possible sequence for the signals described is illustrated below.
   {name: 'otp_lc_data_o.valid',             wave: '0.|...|.1.|...|...'},
   {name: 'otp_lc_data_o.state',             wave: '03|...|...|...|...'},
   {name: 'otp_lc_data_o.count',             wave: '03|...|...|...|...'},
+  {}
```

Comment:
o is this just to create space between the waves? i didnt even know you could do that.

Reply:
Exactly, it is useful if you want to group signals together.